rules-reviewed/eap6/commonj/commonj.windup.xml

```diff
@@ -12,7 +12,7 @@
       </dependencies>
       <sourceTechnology id="weblogic" />
       <sourceTechnology id="websphere" />
-      <targetTechnology id="eap" versionRange="[6,)" />
+      <targetTechnology id="eap" versionRange="[6,7)" />
```

Comment:
indeed, I just commented the first change as I noticed you changed all similar rules version ranges ;)